app/frontend/.env.development

```diff
@@ -1,4 +1,4 @@
-REACT_APP_API_BASE_URL=http://localhost:8888
+REACT_APP_API_BASE_URL=https://dev-api.coucher.org
```

Comment:
Since these are dev environment variables, they should be pointing to the local API rather than the dev preview API. Also some of us prefer to use a local copy of the API and I rather not have to change this back each time.

It's fine if you don't want to set up the Docker stuff, you can still change this locally on your machine but not commit this. With all that said, can you revert this change please?

Reply:
Yep good catch